src/tools/annotation/FreehandRoiTool.js

```diff
@@ -90,6 +90,8 @@ export default class FreehandRoiTool extends BaseAnnotationTool {
     this.selectedToolRoiId = null;
     this.visibleToolRoiIds = [];
     this.onContourRightClicked = null;
+    this.onContourCopyHotKeyPressed = null;
```

Comment:
These names aren't consistent with the ones below

Reply:
37effe391068133391eeb8b8a3f07670a15d963b